packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -347,6 +488,97 @@ Future<void> buildGradleProject({
   }
 }
 
+Future<void> buildGradleAar({
```

Comment:
According to https://codecov.io/gh/flutter/flutter/pull/35217/diff#D21-491, this has no test coverage